apps/src/lib/util/AnalyticsReporter.js

```diff
@@ -48,7 +49,17 @@ class AnalyticsReporter {
 
   sendEvent(eventName, payload) {
     if (this.shouldPutRecord(ALWAYS_SEND)) {
-      track(eventName, payload);
+      if (!eventName) {
+        logToCloud.addPageAction(
+          logToCloud.PageAction.NoValidEventNameLogError,
+          {
+            payload: payload
+          }
+        );
+        track('NO_VALID_EVENT_NAME_LOG_ERROR', payload);
```

Comment:
This logs an unexpected Amplitude event so it will be obvious when this is triggered, telling us to go check NewRelic for the more detailed info about where to fix it.